BlockSettleUILib/LoginWindow.cpp

```diff
@@ -21,9 +21,17 @@
 #include "ZmqContext.h"
 #include "ui_LoginWindow.h"
 #include "FutureValue.h"
+#include "QBitmap"
 
 namespace {
    const auto kAutheIdTimeout = int(BsClient::autheidLoginTimeout() / std::chrono::seconds(1));
+   const auto kProdTitle = QObject::tr("Login Into BlockSettle");
```

Comment:
From my perspective, it should be either "Login to BlockSettle" or "Log Into BlockSettle".

Reply:
changed to "Login to BlockSettle"